jobs/telemetry_batch_view.py

```diff
@@ -25,15 +25,17 @@ def call_exit_errors(command):
 command = ["sbt", "assembly"]
 call_exit_errors(command)
 
-command = sum([
-    ["--{}".format(key[4:].lower().replace("_", "-")), value]
-    for key, value in environ.items()
-    if key.startswith("TBV_") and key != "TBV_CLASS"
-], [
+command = ["spark-submit"]
+if "TBV_executor_cores" in environ:
```

Comment:
Can options be passed without having to create these special cases?

Reply:
we could rework how options that go before the jar get defined